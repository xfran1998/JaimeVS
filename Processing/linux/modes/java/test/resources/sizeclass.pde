// Thanks StanLepunK: https://github.com/processing/processing4/issues/317
Truc truc = new Truc();

void setup() {
  size(200,200);
  truc.size(1,1); // problem >>> error Ã  "."
  // func();
}

void draw() {
  truc.size(1,1); // no problem
}

void func() {
  truc.size(1,1); // no problem
}

class Truc {
  void size(int x, int y) {
  }
}
