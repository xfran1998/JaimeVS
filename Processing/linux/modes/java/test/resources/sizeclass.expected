import processing.core.*;
import processing.data.*;
import processing.event.*;
import processing.opengl.*;

import java.util.HashMap;
import java.util.ArrayList;
import java.io.File;
import java.io.BufferedReader;
import java.io.PrintWriter;
import java.io.InputStream;
import java.io.OutputStream;
import java.io.IOException;

public class sizeclass extends PApplet {

// Thanks StanLepunK: https://github.com/processing/processing4/issues/317
Truc truc = new Truc();

 public void setup() {
  /* size commented out by preprocessor */;
  truc.size(1,1); // problem >>> error \u00e0 "."
  // func();
}

 public void draw() {
  truc.size(1,1); // no problem
}

 public void func() {
  truc.size(1,1); // no problem
}

class Truc {
   public void size(int x, int y) {
  }
}


    public void settings() { size(200, 200); }

    static public void main(String[] passedArgs) {
        String[] appletArgs = new String[] { "sizeclass" };
        if (passedArgs != null) {
            PApplet.main(concat(appletArgs, passedArgs));
        } else {
            PApplet.main(appletArgs);
        }
    }
}
